### DISCUSSION TITLE
Unexpected Link for downloading data

### DISCUSSION THREAD

**Discussion by xu-ziwei** [Sentinal4D/cellshape#12] [Q&A] [answered]
While exploring the "Data Availability" section of this project, I clicked on a link expecting to find the relevant data. However, I was redirected to a site that appeared to be unrelated to the project or its data.

If anyone has insights or can point me to the correct data source, it would be greatly appreciated.

**Comment by DeVriesMatt** [comment, marked as answer]
Hi, thanks for bringing this to my attention. The data was originally published on Zenodo Sandbox, however, I now see the error message "Heads up: The sandbox system is currently unstable due to ongoing testing of the underlying infrastructure.". 

We are working on making the data available on a public data store and will update the project website with this as soon as it is done. 

Sorry about that!
